transport/ax25/ax25.go

```diff
@@ -120,14 +121,25 @@ type Dialer struct {
 }
 
 func (d Dialer) DialURL(url *transport.URL) (net.Conn, error) {
+	return d.DialURLContext(context.Background(), url)
+}
+
+func (d Dialer) DialURLContext(ctx context.Context, url *transport.URL) (net.Conn, error) {
```

Comment:
Should the godoc from `ardop/dial.go` be here too? Sure is nice to have in IDEs that can show godoc. Same comment applies in other files.